qa/support/logstash-integration-failure_injector/logstash-integration-failure_injector.gemspec

```diff
@@ -0,0 +1,27 @@
+
+Gem::Specification.new do |s|
+  s.name            = "logstash-integration-failure_injector"
+  s.version         = "0.0.1"
+  s.licenses        = ["Apache-2.0"]
+  s.summary         = "A collection of Logstash plugins that halp simulating abnormal cases during the tests."
+  s.description     = "This gem is a Logstash plugin required to be installed on top of the Logstash core pipeline using $LS_HOME/bin/logstash-plugin install gemname."
+  s.authors         = ["Elastic"]
+  s.email           = "info@elastic.co"
+  s.homepage        = "https://www.elastic.co/logstash"
+  s.metadata        = {
+    "logstash_plugin" => "true",
+    "logstash_group" => "integration",
+    "integration_plugins" => %w(
+      logstash-input-failure_injector
```

Comment:
```suggestion
```

slow start and plugin crash cases can be simulated with either filter or output.